tests/features/text_formats.feature

```diff
@@ -0,0 +1,29 @@
+@api
+Feature: Text formats configuration
+  In order to input text in the website
+  As a user
+  I need specific text formats to be available and correctly configured
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+
+  @api
+  Scenario: Text formats should be available
+    When I go to "admin/config/content/formats"
+    Then I should see "Full HTML"
+    And I should see "Filtered HTML"
+    And I should see "Basic HTML"
+    And I should see "Plain text"
+  
+  @api
+  Scenario: Text formats should be correctly configured
```

Comment:
I'd rather like to see a scenario from the standpoint of the editor and the behaviour he expects. The current scenario just does a one-on-one comparison of what's supplied in the hook_filter_default_formats() implementation and the corresponding setting as viewed in the admin UI. We do not test the actual behavior, which is that a <abbr> tag including its title - will be rendered when an editor inserts it in a node body field with one of those filter formats and then views the node.